public/doc/openapi-3-v3.0.json

```diff
@@ -1914,6 +1914,15 @@
       "Source": {
         "type": "object",
         "properties": {
+          "app_creation_workflow": {
+            "type": "string",
+            "readOnly": true,
```

Comment:
should it be read only? customer should be able to change his/her mind and switch it, shouldn't (s)he?

Reply:
Ahhh yeah thats a good point. I was just thinking on initial source creation they would pick the workflow and their applications and be done. They could come back and be like "I don't really feel like doing the wizard again, do it for me".

I'll get that changed.